docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+sharelatex:
+  image: sharelatex/sharelatex
+  links:
+    - mongo
+    - redis
+  ports:
+    - "80:80"
```

Comment:
I have the same feeling as you about the interest for a contribution in this project, so...
In fact, it would be better to change the port 5000 for 80 :) But I think it's important to be consistent with the command in the documentation :)

Anyway, thank for your nice job :)